### PR TITLE
Core Team Minutes 5/3/19

### PR DESCRIPTION
**Pull request by BKiddo**

### CODE REVIEW

File: 2018/2019-03-05.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	2. Inviting Sponsors
+	3. Emergency Contact Number
+
+## Minutes
+1. https://github.com/srobo/core-team-minutes/issues
+	1. The hack day was a success, several tasks were assigned and the volcano skeleton has now been built.
+	2. Jake has drafted the email but is being blocked by media consent. Peter will assist with the licensing approval.
+	3. Tyler will speak with Dan and Kajetan to open a conversation with SUSU about competition requirements. We need to begin hiring certain things for the competition soon but require trustee budget approval. 
+	4. Andy BS will work with Kajetan to begin planning a setup schedule with appropriate volunteers assigned to tasks.
+	5. The tech day is only being run in Southampton, and no teams have officially confirmed attendance. If no teams turn up then the day will be run like a hack day in Room 25.
```
Comment (RealOrangeOne) [inline 2018/2019-03-05.md]: minor fix
```suggestion
	5. The tech day is only being run in Southampton, and no teams have officially confirmed attendance. If no teams turn up then the day will be run like a hack day in Building 25.
```

File: 2018/2019-03-05.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+2. 
+	1. Dan will send the proposal to the trustees tomorrow after core team reviews.
+	2. Antoine can invite potential sponsors to attend the competition.
+	3. On the forums Kajetan's phone number was given as a contact. This is problematic in terms of DBS and privacy concerns. We will make it clear that teachers are solely responsible for competitors, and Jake will look into the potential of an emergency specific SIM.  
+
+## Action Points
+### Specific
+- Peter will assist Jake with the media consent licensing approval.
```
Comment (RealOrangeOne) [inline 2018/2019-03-05.md]: The primary assistance is around testing the ticketing process, over the licensing changes.

File: 2018/2019-03-05.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	5. Tech Day
+2. AOB
+	1. Development Team
+	2. Inviting Sponsors
+	3. Emergency Contact Number
+
+## Minutes
+1. https://github.com/srobo/core-team-minutes/issues
+	1. The hack day was a success, several tasks were assigned and the volcano skeleton has now been built.
+	2. Jake has drafted the email but is being blocked by media consent. Peter will assist with the licensing approval.
```
Comment (RealOrangeOne) [inline 2018/2019-03-05.md]: Related to below. The primary assistance is around testing the ticketing process, over the licensing changes.

File: 2018/2019-03-05.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	5. Tech Day
+2. AOB
+	1. Development Team
+	2. Inviting Sponsors
+	3. Emergency Contact Number
+
+## Minutes
+1. https://github.com/srobo/core-team-minutes/issues
+	1. The hack day was a success, several tasks were assigned and the volcano skeleton has now been built.
+	2. Jake has drafted the email but is being blocked by media consent. Peter will assist with the ticketing licensing approval.
```
Comment (PeterJCLaw) [inline 2018/2019-03-05.md]: I'm not sure this is quite right -- I don't recall talking about the licensing of the tickets; we did talk about the licensing of the media consent form though.

I think the thing I was going to help with was checking that the media consent & tickets systems _work_ (https://github.com/srobo/tasks/issues/61).

**Pull request merged**